news/2024/2024-11-15-osulazer-updates-november-15-2024.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "osu!(lazer) Updates: November 15, 2024"
+date: 2024-11-16 23:00:00 +0000
+---
+
+This update contains a lot of quality-of-life changes. Feel free to continue your day.
+
+![](/wiki/shared/news/2024-11-16-osulazer-updates-november-15-2024/banner.jpg)
+
+For those who want to pause for a moment to read, here's the complete rundown:
+
+After the [last update](https://osu.ppy.sh/home/news/2024-10-09-osulazer-updates-october-9), an [update to performance algorithms](https://osu.ppy.sh/home/news/2024-10-28-performance-points-star-rating-updates) was rolled out to all players. If your global rank went up, congratulations. If not, maybe it's time to get back on the grind.
+
+Below you will find [peppy](https://osu.ppy.sh/users/2) and [pishifat](https://osu.ppy.sh/users/3178418) discussing the rollout as well as all the new neat changes in the game.
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/YKljRdxfhxQ" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Editor changes
+
+- **Added keyboard shortcut for cycling grid types (`Shift` + `G`).** Did you know we have not just square, but triangular and circular grids as well?
```

Comment:
```suggestion
- **Added a keyboard shortcut for cycling grid types (`Shift` + `G`).** Did you know we have not just square, but triangular and circular grids as well?
```

Reply:
it's actually supposed to be a noun phrase (and not a sentence), i.e. "added" is an adjective (but it's ambiguous and works both ways)